spec/models/comment_spec.rb

```diff
@@ -1,5 +1,32 @@
-require 'rails_helper'
+require "rails_helper"
 
 RSpec.describe Comment, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  let(:user) { build(:user) }
+  subject { build(:comment, user: user) }
+
+  context "when the factory is build" do
+    it { is_expected.to be_valid }
+    it { is_expected.to be_a Comment }
+  end
+
+  context "when calling the instance methods" do
+    it do
+      is_expected.to respond_to(
+        :user_id, :content, :comment_on_id, :comment_on_type, :votes_count,
+        :user, :comment_on, :votes
+      )
+    end
+  end
+
+  describe "ActiveModel Validations" do
+    it { is_expected.to validate_presence_of(:user) }
+    it { is_expected.to validate_presence_of(:content) }
+  end
+
+  describe  "ActiveModel Relation" do
+    it { is_expected.to belong_to(:user) }
+    it { is_expected.to belong_to(:comment_on) }
+  end
+
+  it_behaves_like "a votable", :comment_with_votes
```

Comment:
Hmm. Did you notice that the saving and retrieval of comments are not being saved. Something like:
> expect Comment.all to be empty
> create Comment
> expect Comment.all not to be empty
> expect Comment.all to size to be 1
> expect Comment.all.first to eql comment

We could also test individual associations
> Create comment for a question
> Expect it returns the question 
> Expect it doesn't return an answer
> Do same for comment on answer
> You can also check for size